config.h.in.cmake

```diff
@@ -203,6 +203,7 @@
 
 #cmakedefine LLVM_BUILD_MODE_DEBUG
 
+#cmakedefine USE_LLVM_FOR_DYNLIBS
```

Comment:
I think this is not necessary since this variable is only used in CMake. I'll remove it in the PR with the file/thread support.